lib/profile/cli.rb

```diff
@@ -109,7 +109,7 @@ def cli_syntax(command, args_str = nil)
     end
 
     command :prepare do |c|
-      cli_syntax(c, 'TYPE')
+      cli_syntax(c, '[TYPE]')
```

Comment:
Please could you also add an option to force the cluster type menu option? Something like `--include-types`.